lib/_http_outgoing.js

```diff
@@ -778,8 +782,8 @@ OutgoingMessage.prototype._flush = function _flush() {
     if (this.finished) {
       // This is a queue to the server or client to bring in the next this.
       this._finish();
-    } else if (ret) {
-      // This is necessary to prevent https from breaking
```

Comment:
Should we keep the comment?

Reply:
@mcollina: I don't think it's relevant. We should be emitting drain regardless.